lib/tasks/populate_per_confirmation_with_responded_by.rake

```diff
@@ -0,0 +1,18 @@
+GITHUB_FRAMEWORK_URI = 'https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git'.freeze
```

Comment:
For my benefit what are lines 1-3 doing?

Reply:
Copy pasta :D I'll remove them